### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Manage Access Policies _Keyboard:When hit on enter button keyboard focus does not go to Id edit box, it remains on Add button.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2099]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS2.1.1;#Keyboard; #Win32;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
5. Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
6.Check whether the keyboard focus goes to the added row after click on Add button.
**​Actual:​**
When hit on enter button keyboard focus does not go to Id edit box, it remains on Add button.

**Expected:**
Keyboard focus should go to the added row after click on Add button.
**UserImpact:**
Keyboard user unable to get the updated data after click on Add button.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
The focus goes to the newly added row .So,the issue is no longer reproduce. Hence close this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
